src/debugger.ts

```diff
@@ -185,7 +186,13 @@ export async function getDebugConfigurationFromCache(cache: CMakeCache, target:
   const debugger_name = platform == 'darwin' ? 'lldb' : 'gdb';
   const description = DEBUG_GEN[debugger_name];
   const gcc_compiler_regex = /([cg]\+\+|g?cc)(?=[^\/\\]*$)/gi;
-  const gdb_debugger_path = debuggerPathOverride || compiler_path.replace(gcc_compiler_regex, description.miMode);
+  let gdb_debugger_path = debuggerPathOverride || compiler_path.replace(gcc_compiler_regex, description.miMode);
+  if (path.isAbsolute(gdb_debugger_path) && !await fs.exists(gdb_debugger_path)) {
+    gdb_debugger_path = path.join(path.dirname(compiler_path), description.miMode);
+    if (process.platform === 'win32') {
+      gdb_debugger_path = gdb_debugger_path.replace('\\', '/') + '.exe';
```

Comment:
Is there a need to flip the slashes direction?

Reply:
Any suggestion? should I remove the replace or just preserve it?

Reply:
Theoretically `path.join()` should do the thing (there's whole module `paths`, if not). But IMO you've missed at least the case sensitivity here.

Reply:
What's the case sensitivity about?

Reply:
I've seen that for win32 we go lowercase sometimes (for example see `util.normalizePath`). Maybe it can be useful here as well.

Reply:
I don't think there is any case-sensitivity problem here. That only applies for linux.  Can you remove the part that flips the slashes for `win32`?  I can approve this PR after that.

Reply:
done